### PR TITLE
Fix permision on the pid file. Adds more compilation properties

### PR DESCRIPTION
**Pull request by DiegoVazquezNanini**
Small set of fixes for the role

@TBonnin @jfsoul

### CODE REVIEW

File: roles/nginx+pagespeed/files/nginx.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 user                nginx;
 worker_processes    auto;
 error_log           /var/log/nginx/error.log;
-pid                 /var/run/nginx.pid;
+pid                 /var/run/nginx/nginx.pid;
```
Comment (TBonnin) [inline roles/nginx+pagespeed/files/nginx.conf]: why this change?

**Comment by DiegoVazquezNanini** [comment]
I do need to log file configs con the compilation and I also need to remove the nginx users from the systemd definition.
